ec/src/models/twisted_edwards_extended.rs

```diff
@@ -296,7 +293,7 @@ mod group_impl {
 
 //////////////////////////////////////////////////////////////////////////////
 
-/// `GroupProjective` implements Extended Twisted Edwards Coordinates
+/// `GroupProjective` implements Extended Twisted Edwards (Jacobian) Coordinates
```

Comment:
```suggestion
/// `GroupProjective` implements Extended Twisted Edwards Coordinates
```

Reply:
It says "Jacobian" coordinates for the SW `GroupProjective`. Are they not the same for TE?

Reply:
No, the non-affine struct for Twisted Edwards is not using the jacobian model. See [here](https://iacr.org/archive/asiacrypt2008/53500329/53500329.pdf) for details.

For SW curves, the `Projective` part is actually a bit of a misnomer; the Jacobian coordinate system is a bit different to the projective system; compare [this](https://en.wikibooks.org/wiki/Cryptography/Prime_Curve/Standard_Projective_Coordinates) with [this](https://en.wikibooks.org/wiki/Cryptography/Prime_Curve/Jacobian_Coordinates).